src/main/java/nextstep/courses/domain/Cardinal.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.courses.domain;
+
+public class Cardinal {
+    private Long id;
```

Comment:
```suggestion
    private final Long id;
```
인스턴스 변수는 가능하면 final로 구현해보고, 변경 가능한 부분에 대해서만 final을 제거하는 방식으로 구현하는 것은 어떨까?